org.eclipse.mylyn-target/mylyn-e4.26.target

```diff
@@ -12,7 +12,7 @@
 			<unit id="org.eclipse.sdk.feature.group" version="0.0.0"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/egit/updates-6.4/"/>
+			<repository location="https://download.eclipse.org/egit/updates-6.2/"/>
```

Comment:
Why do we need to degrade?

Reply:
I have fixed this. wait for the next commit.